recipe/meta.yaml

```diff
@@ -1,63 +1,68 @@
-{% set version = "0.1.5.post2" %}
-{% set lib_ver = version.split(".")[:3] | join(".") %}
+{% set name = "qdldl-python" %}
+{% set version = "0.1.7" %}
+{% set pypiname = "qdldl" %}
 
 package:
-  name: qdldl-python
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/osqp/qdldl-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 8825a2f4aff86cbac5f2a9783bde9a6a454ab3937d34ae18ed0affaba775ed8c
+  url: https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz
+  sha256: 6c86639d11e301e78af2cce885249a286f675cce96d0e6d60f5b14f859d0f10e
   patches:
-    - patches/0001-use-Ninja-everywhere.patch
-    - patches/0002-build-against-conda-forge-s-libqdldl.patch
+    - patches/0001-use-Ninja-and-add_CMAKE_INSTALL_PREFIX.patch
+    - patches/0002-build-against-external-libqdldl.patch
     - patches/0003-build-qdldlamd-as-shared-lib-and-install-it.patch
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
     - ninja
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - pip
-    - libqdldl ={{ lib_ver }}
-    - numpy
-    - pybind11
+    - setuptools
+    - wheel
+    - libqdldl {{ version }}
+    - numpy {{ numpy  }}
+    - pybind11 2.10.1
   run:
     - python
+    - numpy >=1.7
     - {{ pin_compatible('numpy') }}
```

Comment:
This can be removed, no?

Reply:
Yeah, just for completeness and to respect the upstream. i.e. use whatever version that is compatible but not less than 1.7.